src/engine/openspaceengine.cpp

```diff
@@ -146,6 +162,19 @@ OpenSpaceEngine::OpenSpaceEngine()
 
     SpiceManager::initialize();
     TransformationManager::initialize();
+
+    addProperty(_printEvents);
+    addProperty(_visibility);
+    addProperty(_showHiddenSceneGraphNodes);
+
+    using Visibility = openspace::properties::Property::Visibility;
+    _visibility.addOptions({
+        { static_cast<int>(Visibility::NoviceUser), "Novice User"},
```

Comment:
veery nitpicky, but missing space before closing brackets for all of the options